thoth/storages/graph/models.py

```diff
@@ -262,6 +263,11 @@ class PackageExtractRun(Base, BaseExtension):
         "ExternalSoftwareEnvironment", back_populates="external_package_extract_runs"
     )
 
+    __table_args__ = (
+        Index("os_id_idx", "os_id"),
+        Index("os_version_id_idx", "os_id"),
```

Comment:
Mind the index is duplicate - 2 indexes with defferent names introduced twice for the same column `os_id`.

To hit the index with query properly, there will be needed one more index with `os_id` and `os_version_id`.